internal/testrunner/runners/system/tester.go

```diff
@@ -483,7 +483,12 @@ func (r *tester) createServiceInfo() (servicedeployer.ServiceInfo, error) {
 	svcInfo.Name = r.testFolder.Package
 	svcInfo.Logs.Folder.Local = r.locationManager.ServiceLogDir()
 	svcInfo.Logs.Folder.Agent = ServiceLogsAgentDir
-	svcInfo.Test.RunID = common.CreateTestRunID()
+
+	prefix := ""
+	if v, found := os.LookupEnv("ELASTIC_PACKAGE_PREFIX_SERVICE_TEST_RUN_ID"); found && v != "" {
+		prefix = v
+	}
+	svcInfo.Test.RunID = common.CreateTestRunIDWithPrefix(prefix)
```

Comment:
In case of using the terraform service deployer, a part of updating the RunID used in as terraform variable, the alias used to render the policy templates in tests needs to be set to the same value as the terraform variable. This is needed for instance for `aws.ec2_metrics` that filters by the some resource name.